src/commands.rs

```diff
@@ -1615,44 +1616,64 @@ struct BackoutArgs {
     destination: Vec<String>,
 }
 
-/// Create, update, or delete a branch
+/// Manage branches.
 ///
 /// For information about branches, see
 /// https://github.com/martinvonz/jj/blob/main/docs/branches.md.
-#[derive(clap::Args, Clone, Debug)]
-struct BranchArgs {
-    /// The branch's target revision
-    #[clap(long, short, group = "action")]
-    revision: Option<String>,
-
-    /// Allow moving the branch backwards or sideways
-    #[clap(long, requires = "revision")]
-    allow_backwards: bool,
-
-    /// Delete the branch locally
+#[derive(clap::Subcommand, Clone, Debug)]
+enum BranchSubcommand {
+    /// Create a new branch.
+    #[clap(visible_alias("c"))]
+    Create {
+        /// The branch's target revision.
+        #[clap(long, short)]
+        revision: Option<String>,
+
+        /// The branches to create.
+        names: Vec<String>,
```

Comment:
I think you can add `#[clap(required = true)]` to have clap check that at least one branch was provided. Same thing on a few others below.